csharp/ql/src/Diagnostics/CompilerError.qhelp

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>
+This is an internal query that finds all compilation errors reported by the C# compiler.</p>
+</overview>
+
+<recommendation>
+<p>
+Fix the cause of the compilation error.
+</p>
+</recommendation>
```

Comment:
I think we should remove the `recommendation` block; such errors should only happen when we are unable to compile code that the build itself is able to compile, i.e., the extractor is running an old version of Roslyn.